google/resourcemanager_helpers.go

```diff
@@ -0,0 +1,14 @@
+package google
+
+import (
+	"fmt"
+
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+func getResourceName(resourceId *cloudresourcemanager.ResourceId) string {
```

Comment:
I don't see any usage for this method?

Reply:
Hmm, this code must have come from something else I was noodling on. Removed those 2 source files.